functions/text-transform/1.0/index.js

```diff
@@ -0,0 +1,31 @@
+const textTransform = async ({
+  value,
+  downcase,
+  upcase,
+  capitalize,
+  parameterize,
+}) => {
+  const transformedValue = parameterize
+    ? parameterized(value)
+    : downcase
+    ? value.toLowerCase()
+    : upcase
+    ? value.toUpperCase()
+    : capitalize
+    ? capitalized(value)
+    : value;
+
+  return { result: transformedValue };
+};
+
+const capitalized = (str) =>
```

Comment:
Nitpicking but maybe we should call this `value` instead of `str` (or the other way around) so it's consitent.